examples/ceed/ex2.h

```diff
@@ -111,11 +111,11 @@ CEED_QFUNCTION(f_apply_diff)(void *ctx, const CeedInt Q,
       // Stored in Voigt convention
       // 0 2
       // 2 1
-      const CeedInt idx[] = {0, 2, 2, 1};
+      const CeedInt idx[2][2] = {{0, 2}, {2, 1}};
       CeedScalar dXdxdXdxT[2][2];
       for (CeedInt m=0; m<2; m++)
         for (CeedInt n=0; n<2; n++)
-          dXdxdXdxT[m][n] = qdata[i+idx[n+2*m]*Q];
+          dXdxdXdxT[m][n] = qdata[i+idx[m][n]*Q];
```

Comment:
With the 2D array indices it reads a bit better, but it still looks to me more time consuming to read/debug/write this than the previous version